feature/session/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -57,453 +57,474 @@
             android:visibility="invisible"
             />
 
-        <androidx.core.widget.NestedScrollView
+        <LinearLayout
+            android:id="@+id/toolbar_parent"
```

Comment:
not only toolbar 👀 Should this have the id anyway?

Reply:
I fixed it.
I renamed to `content_parent`